protocol/contracts/beanstalk/storage/System.sol

```diff
@@ -316,7 +316,7 @@ struct AssetSettings {
     int96 milestoneStem; //                 │ 12 (30)
     bytes1 encodeType; //                   │ 1  (31)
     // one byte is left here.             ──┘ 1  (32)
-    int24 deltaStalkEarnedPerSeason; // ────┐ 3
+    int32 deltaStalkEarnedPerSeason; // ────┐ 3
```

Comment:
the comments should be updated based on this (i.e, this is now 4 bytes instead of 3), and the summation is now different.

Reply:
Fixed here: https://github.com/BeanstalkFarms/Beanstalk/pull/1092/commits/9249efddefd2f3013c10e468ab03c4b4b1e91ad9#diff-d66284fdb000ab9862246d20e3858bd1753a53ec3b03fde839ee1c5245edf4f4R319